src/Microsoft.DotNet.Files/ContentFiles.cs

```diff
@@ -19,45 +20,68 @@ public ContentFiles(ProjectContext context)
             _context = context;
         }
 
-        public void StructuredCopyTo(string targetDirectory)
+        public void StructuredCopyTo(string targetDirectory, IEnumerable<IncludeEntry> includeEntries)
         {
-            var sourceFiles = _context
-                .ProjectFile
-                .Files
-                .GetContentFiles();
+            if (includeEntries == null)
```

Comment:
Unless you are removing it now, separate it in two methods. It is kind of hard to read it as it is, plus, introducing two methods should be very costless.